modules/bending_energy_diffusion_loss_notes.ipynb

```diff
@@ -0,0 +1,644 @@
+{
```

Comment:
<p>Didn't see non-normalized in this figure?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1589/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
They overlap. I will mention this in the text.